src/shim/runtime/v2/rune/v2/create.go

```diff
@@ -0,0 +1,140 @@
+package v2
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/confidential-containers/enclave-cc/src/shim/runtime/v2/rune/config"
+	"github.com/confidential-containers/enclave-cc/src/shim/runtime/v2/rune/oci"
+	shimtypes "github.com/confidential-containers/enclave-cc/src/shim/runtime/v2/rune/types"
+	"github.com/containerd/containerd/api/types"
+	"github.com/containerd/containerd/runtime/v2/runc"
+	taskAPI "github.com/containerd/containerd/runtime/v2/task"
+	"github.com/sirupsen/logrus"
+)
+
+var ContainerBase = "/run/enclave-cc/app"
+
+func create(ctx context.Context, s *service, r *taskAPI.CreateTaskRequest) (*runc.Container, error) {
+	ociSpec, err := config.LoadSpec(filepath.Join(r.Bundle, configFilename))
+	if err != nil {
+		return nil, err
+	}
+	containerType, err := oci.ContainerType(*ociSpec)
+	if err != nil {
+		return nil, err
+	}
+	sandboxNamspace, err := oci.SandboxNamespace(*ociSpec)
+	if err != nil {
+		return nil, err
+	}
+
+	var container *runc.Container
+
+	switch containerType {
+	case shimtypes.PodSandbox:
```

Comment:
These `case` blocks are too big imho. Could you refactor to call `handlePodSandbox()`, `handlePodContainer()`, _etc_?